ppx/test/pp.expected

```diff
@@ -0,0 +1,736 @@
+module type Foo  =
```

Comment:
This file should be removed (was moved to pp_reason.expected), prob a bad merge.